corehq/apps/hqwebapp/views.py

```diff
@@ -500,19 +500,21 @@ def logout(req, default_domain_redirect='domain_login'):
 @location_safe
 @two_factor_exempt
 def ping_login(request):
-    return JsonResponse({
-        'success': request.user.is_authenticated,
-        'last_request': request.session.get('last_request'),
-        'username': request.user.username,
-    })
+    return _ping_response(request)
```

Comment:
Since these two views are identical, you could also keep just one and maintain two different entries in `urls.py`, both pointing to that.

Reply:
good point, 5bfbb52f8cc